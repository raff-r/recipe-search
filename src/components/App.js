import React, { Component } from 'react';

import SearchBar from './search-bar';

class App extends Component {
  render() {
    return (
      <div className="container-fluid">
        <div className="row">
          <div className="col-xs-12">
            <SearchBar />
          </div>
        </div>
      </div>
    );
  }
}

export default App;
