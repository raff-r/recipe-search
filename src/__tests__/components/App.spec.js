import React from 'react';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";

import App from '../../components/App';

describe('SearchResults Component', () => {

    let component;

    beforeEach(() => {
        component = shallow( <App /> );
    });

    it('should match the snapshot', () => {
        expect(toJson(component)).toMatchSnapshot();
    });

});