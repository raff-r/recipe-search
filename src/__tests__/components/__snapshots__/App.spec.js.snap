// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults Component should match the snapshot 1`] = `
<div
  className="container-fluid full-height"
>
  <div
    className="row full-height"
  >
    <div
      className="col-xs-12 full-height"
    >
      <Connect(SearchBar) />
      <Connect(SearchResults) />
    </div>
  </div>
</div>
`;
